steps/post-jira-comment-with-build-details-step/0.1.0/step.yml

```diff
@@ -0,0 +1,72 @@
+title: Create JIRA comment with build
+summary: Will post a comment on the issue page with build details (download link,
```

Comment:
Step's description will be visible on bitrise.io. Please provide a description as well.